bindings/nodejs/lib/types/wallet/transaction-options.ts

```diff
@@ -4,45 +4,63 @@
 import { TaggedDataPayload } from '../block/payload/tagged';
 import { Burn } from '../client';
 
-/** Options for the transaction creation */
+/** Options for the transaction creation. */
 export interface TransactionOptions {
+    /** The strategy applied for base coin remainders. */
     remainderValueStrategy?: RemainderValueStrategy;
+    /** An optional tagged data payload. */
     taggedDataPayload?: TaggedDataPayload;
-    /** Custom inputs that should be used for the transaction */
+    /** 
+     * Custom inputs that should be used for the transaction.
+     * If custom inputs are provided only those are used. If also other additional inputs should be used, `mandatory_inputs` should be used instead.
+     */
     customInputs?: string[];
+    /** Inputs that must be used for the transaction. */
     mandatoryInputs?: string[];
+    /** Specifies what needs to be burned during input selection. */
     burn?: Burn;
-    /** Optional note, that is only stored locally */
+    /** Optional note, that is only stored locally. */
     note?: string;
+    /** Whether to allow sending a micro amount. */
     allowMicroAmount: boolean;
 }
 
-/** The RemainderValueStrategy */
+/** The possible remainder value strategies. */
 export type RemainderValueStrategy =
     | ChangeAddress
     | ReuseAddress
     | CustomAddress;
 
-/** ChangeAddress variant of RemainderValueStrategy */
+/**
+ * Allows to move the remainder value to a change address.
+ */
 export type ChangeAddress = {
+    /** The name of the strategy. */
     strategy: 'ChangeAddress';
+    /** TODO */
```

Comment:
I think this field was just required to make it compatible with `CustomAddress` where the field can actually be something